Makefile

```diff
@@ -0,0 +1,21 @@
+# All documents to be used in spell check.
+ALL_DOC := $(shell find . -name '*.md' -type f | sort)
+
+MISSPELL=misspell -error
+MISSPELL_CORRECTION=misspell -w
+
+.PHONY: circle-ci
+circle-ci: misspell
+
+.PHONY: misspell
+misspell:
+	$(MISSPELL) $(ALL_DOC)
+
+.PHONY: misspell-correction
+misspell-correction:
+	$(MISSPELL_CORRECTION) $(ALL_DOC)
+
+.PHONY: install-tools
+install-tools:
+	GO111MODULE=on go install \
+	  github.com/client9/misspell/cmd/misspell
```

Comment:
Use this [approach](https://github.com/open-telemetry/opentelemetry-go/blob/623b062e17fb1aebae74aad74d1fc637bde520b2/Makefile#L19-L20) to install tools locally to guarantee that same version is used.

Reply:
Thanks, updated in https://github.com/open-telemetry/rfcs/pull/14/commits/1d30f395916a8c5070c091e1a2df6b07901cd389.